api/spec/requests/api/v1/comments_spec.rb

```diff
@@ -1,4 +1,3 @@
-require "rails_helper"
```

Comment:
Rails helper is included in the swagger_helper, so I don't think it's needed here